src/main/kotlin/no/nav/syfo/personoppgavehendelse/domain/PersonoppgavehendelseType.kt

```diff
@@ -7,4 +7,6 @@ enum class PersonoppgavehendelseType {
     DIALOGMOTESVAR_BEHANDLET,
     MOTEBEHOV_SVAR_MOTTATT,
     MOTEBEHOV_SVAR_BEHANDLET,
```

Comment:
Ser at disse to ikke blir brukt i dag 🤔

Reply:
Synes jeg husker at jeg la det inn sånn at enum'en skulle være konsistent med det som finnes i syfooversiktsrv.